### PR TITLE
Qa 75 production gitlab pipeline

### PR DESCRIPTION
**Pull request by mzedel**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+stages:
+  - build
+  - test
+  - publish
+  - integration
+
+before_script:
+  - docker info
+  - git clone git://github.com/mendersoftware/mendertesting
+  - . mendertesting/check_commits.sh
```
Comment (lluiscampos) [inline .gitlab-ci.yml]: We should execute the script rather than source it. Sourcing it might have strange side effects

Comment (lluiscampos) [reply .gitlab-ci.yml]: I meant: `./mendertesting/check_commits.sh`

Comment (mzedel) [reply .gitlab-ci.yml]: thanks for catching that - I guess I meant to write `./mendertesting/check_commits.sh` - it's now `mendertesting/check_commits.sh` as you proposed + moved to a separate step

**Comment by mzedel** [comment]
thanks for looking into this! I added the acceptance tests and made some adjustments to the commit checking...

**Pull request merged**